### DISCUSSION TITLE
Compiling on Debian

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#1891] [General]
I couldn't stand anymore using  version 0.14.1 on my laptop which I have to use often, so I first had to upgrade it to bullseye, due to old qt version on stable (buster).

As there is still missing a debian maintainer for LXQt I compiled all [following the wiki](https://github.com/lxqt/lxqt/wiki/Building-from-source) and all went well (2 packages were missing, added to wiki - and sure it was slow but not so bad as I thought).

But as it wasn't a clean system there is some issue left: no trash support (only "remove" in context menu), no trashbin neither on desktop nor in sidebar, and all mounted devices are always shown in sidebar (/, /tmp, /home) which isn't like that on arch. 

Writing this it came to my mind that gvfs could miss, and in fact it was. So all solved ;)

**Comment by tsujan** [comment]
When I used Debian, I'd prepared debian folders for all LXQt components. I used them to make deb packages, as I use PKGBUILD now. I set the version to the compilation date.

That method had several advantages over installation from sources: I could control the packages with `apt`, especially their dependencies; I had a single package for each component (no splitting into dev packages); system updates didn't affect my installation (because of package versions); etc.
